src/openms/source/FORMAT/HANDLERS/MzIdentMLDOMHandler.cpp

```diff
@@ -957,12 +957,16 @@ namespace OpenMS
                       }
                       else if (spci->second.front().getAccession() == "MS:1002057")  // pro nterm
                       {
-                        ResidueModification m = ModificationsDB::getInstance()->getModification(mname,  r, ResidueModification::PROTEIN_N_TERM);
+                        // TODO: add support for protein N-terminal modifications in unimod
```

Comment:
"in unimod" makes it sound like Unimod lacks support for this. Better "from Unimod"?